<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SchwarzDialog</class>
 <widget class="QDialog" name="SchwarzDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="radiusLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Radius:</string>
   </property>
  </widget>
  <widget class="QLabel" name="heightLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Height:</string>
   </property>
  </widget>
  <widget class="QLabel" name="nLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Cross section verts:</string>
   </property>
  </widget>
  <widget class="QLabel" name="mLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of layers:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="radiusLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="heightLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLabel" name="angleLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Turn angle:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="angleLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0.314159</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkbox_openness">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>112</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Check if open</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkbox_springs">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>191</y>
     <width>223</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Check if elastic face connection</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SchwarzDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SchwarzDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
